README.md

```diff
@@ -109,3 +109,7 @@ To see the apps bound to a service:
 `cf services`
 
 [user provided service]: https://docs.cloudfoundry.org/devguide/services/user-provided.html#credentials
+
+### Routes
+
+If an app is bound that doesn't have a route or if an app that's already bound has all its routes removed prometheus will not be able to scrape it. This will be recorded in the `up()` metric
```

Comment:
very nitpicky, but parentheses don't really exist in metric names. I think this would be more accurately listed as `up` rather than `up()`